HypothesisSetup.asciidoc

```diff
@@ -0,0 +1,34 @@
+# Building the fabric8-hypothesis-core-image image
+
+1. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-core-image.git
+2. To change the default node version, npm version, push registry, pull registry, organisation and port number, change the respective values in the constants.sh 
+3. Trigger the build using cico_build_deploy.sh script and wait for pushes to be complete.
+ 
+
+# Building the fabric8-hypothesis-manager-api-image image
+
+1. First make sure that you have built and pushed the fabric8-hypothesis-core-image image with the approriate default node version, npm version, push registry, pull registry, organisation and port values.
+2. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-manager-api-image.git
+* For changing the set_env.sh file location of fabric8-hypothesis-core-image, change the env_script_location value in the app_constants.sh file.
+* For changing the app_name, change the APP_NAME value in the app_constants.sh file.
+* For changing the port, route_path, route_prefix and hostname, change the respective values in the supervisor.env file in the openshift folder.
+3. Trigger the build using cico_build_deploy.sh script and wait for pushes to be complete.
+
+# Building the fabric8-hypothesis-data-api-image image
+
+1. First make sure that you have built and pushed the fabric8-hypothesis-core-image image with the approriate default node version, npm version, push registry, pull registry, organisation and port values.
+2. Fork this repo: https://github.com/fabric8-hdd/fabric8-hypothesis-data-api-image.git
+* For changing the set_env.sh file location of fabric8-hypothesis-core-image, change the env_script_location value in the app_constants.sh file.
+* For changing the app_name, change the APP_NAME value in the app_constants.sh file.
+* For changing the port, route_path, route_prefix and hostname, change the respective values in the data-api.env file in the openshift folder.
+3. Trigger the build using cico_build_deploy.sh script and wait for pushes to be complete.
+
+
+# Building the fabric8-hypothesis-measurements-api-image image
+
+1. First make sure that you have built and pushed the fabric8-hypothesis-core-image image with the approriate default node version, npm version, push registry, pull registry, organisation and port values.
```

Comment:
there's no image for hypothesis-core-image but instead the image built from hypothesis-core-image is nodejs image which is a base layer for other images